### Importación de datos



In [ ]:
import pandas as pd

url = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_1%20.csv"
url2 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_2.csv"
url3 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_3.csv"
url4 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_4.csv"

tienda = pd.read_csv(url)
tienda2 = pd.read_csv(url2)
tienda3 = pd.read_csv(url3)
tienda4 = pd.read_csv(url4)

tienda.head()

,Producto,Categoría del Producto,Precio,Costo de envío,Fecha de Compra,Vendedor,Lugar de Compra,Calificación,Método de pago,Cantidad de cuotas,lat,lon
0,Asistente virtual,Electrónicos,164300.0,6900.0,16/01/2021,Pedro Gomez,Bogotá,4,Tarjeta de crédito,8,4.60971,-74.08175
1,Mesa de comedor,Muebles,192300.0,8400.0,18/05/2022,Beatriz Morales,Medellín,1,Tarjeta de crédito,4,6.25184,-75.56359
2,Juego de mesa,Juguetes,209600.0,15900.0,15/03/2021,Juan Fernandez,Cartagena,1,Tarjeta de crédito,1,10.39972,-75.51444
3,Microondas,Electrodomésticos,757500.0,41000.0,03/05/2022,Juan Fernandez,Cali,4,Nequi,1,3.43722,-76.52250
4,Silla de oficina,Muebles,335200.0,20200.0,07/11/2020,Maria Alfonso,Medellín,5,Nequi,1,6.25184,-75.56359


#1. Análisis de facturación



In [4]:
import pandas as pd

# URLs de tus bases de datos
url = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_1%20.csv"
url2 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_2.csv"
url3 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_3.csv"
url4 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_4.csv"

# Cargar los datos de cada tienda en DataFrames de Pandas
tienda1_df = pd.read_csv(url)
tienda2_df = pd.read_csv(url2)
tienda3_df = pd.read_csv(url3)
tienda4_df = pd.read_csv(url4)

# Diccionario para almacenar los DataFrames y facilitar la iteración
tiendas_data = {
    "Tienda 1": tienda1_df,
    "Tienda 2": tienda2_df,
    "Tienda 3": tienda3_df,
    "Tienda 4": tienda4_df
}

# Diccionario para almacenar el total de facturación de cada tienda
total_facturacion_por_tienda = {}

# Calcular el total de facturación para cada tienda
print("--- Calculando la facturación por tienda ---")
for nombre_tienda, df in tiendas_data.items():
    # Asumo que la columna 'Precio' es la que representa el ingreso por cada venta.
    # Si tienes una columna de 'Cantidad' y 'Precio' por unidad, deberías hacer df['Precio'] * df['Cantidad'].
    # Sin embargo, basándonos en tu código original que suma 'ingreso', aquí sumaremos 'Precio'.
    if 'Precio' in df.columns:
        # Asegurarse de que la columna 'Precio' sea numérica y manejar posibles nulos
        df['Precio'] = pd.to_numeric(df['Precio'], errors='coerce')
        total_facturacion = df['Precio'].sum()
        total_facturacion_por_tienda[nombre_tienda] = total_facturacion
        print(f"Total de facturación para {nombre_tienda}: ${total_facturacion:,.2f}")
    else:
        print(f"Advertencia: La columna 'Precio' no se encontró en los datos de {nombre_tienda}. Verifique el nombre de la columna de ingresos.")
        total_facturacion_por_tienda[nombre_tienda] = 0 # Asignar 0 si no se encuentra la columna

print("\n" + "="*50 + "\n") # Separador para mejor legibilidad

# Filtrar tiendas con facturación para evitar errores si alguna no tiene la columna de ingresos o su suma es 0
if total_facturacion_por_tienda:
    tiendas_con_datos_validos = {k: v for k, v in total_facturacion_por_tienda.items() if v > 0}

    if tiendas_con_datos_validos:
        # Encontrar la tienda con más facturación y la tienda con menos facturación
        tienda_mas_facturacion = max(tiendas_con_datos_validos, key=tiendas_con_datos_validos.get)
        tienda_menos_facturacion = min(tiendas_con_datos_validos, key=tiendas_con_datos_validos.get)

        print(f"La tienda que generó MÁS facturación es: **{tienda_mas_facturacion}** con un total de **${total_facturacion_por_tienda[tienda_mas_facturacion]:,.2f}**")
        print(f"La tienda que generó MENOS facturación es: **{tienda_menos_facturacion}** con un total de **${total_facturacion_por_tienda[tienda_menos_facturacion]:,.2f}**")

        # Calcular la facturación total de las 4 tiendas
        facturacion_total_general = sum(total_facturacion_por_tienda.values())
        print(f"\nLa **Facturación Total General** de las 4 tiendas es: **${facturacion_total_general:,.2f}**")

    else:
        print("No se pudo calcular la facturación para ninguna tienda válida (todos los ingresos son cero o no hay datos).")
else:
    print("No se encontraron datos de facturación para analizar.")


# Opcional: Visualización con Pandas para un resumen tabular
print("\n" + "="*50 + "\n") # Separador
print("Resumen de Facturación (Tabla):")
df_facturacion = pd.DataFrame(list(total_facturacion_por_tienda.items()), columns=['Tienda', 'Total Facturación'])
print(df_facturacion.sort_values(by='Total Facturación', ascending=False).to_string(index=False))

--- Calculando la facturación por tienda ---
Total de facturación para Tienda 1: $1,150,880,400.00
Total de facturación para Tienda 2: $1,116,343,500.00
Total de facturación para Tienda 3: $1,098,019,600.00
Total de facturación para Tienda 4: $1,038,375,700.00


La tienda que generó MÁS facturación es: **Tienda 1** con un total de **$1,150,880,400.00**
La tienda que generó MENOS facturación es: **Tienda 4** con un total de **$1,038,375,700.00**

La **Facturación Total General** de las 4 tiendas es: **$4,403,619,200.00**


Resumen de Facturación (Tabla):
  Tienda  Total Facturación
Tienda 1       1150880400.0
Tienda 2       1116343500.0
Tienda 3       1098019600.0
Tienda 4       1038375700.0


# 2. Ventas por categoría

In [6]:
import pandas as pd

# URLs de tus bases de datos
url = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_1%20.csv"
url2 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_2.csv"
url3 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_3.csv"
url4 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_4.csv"

# Cargar los datos de cada tienda en DataFrames de Pandas
tienda1_df = pd.read_csv(url)
tienda2_df = pd.read_csv(url2)
tienda3_df = pd.read_csv(url3)
tienda4_df = pd.read_csv(url4)

# Lista para almacenar todos los DataFrames
lista_de_dfs = [tienda1_df, tienda2_df, tienda3_df, tienda4_df]

# Concatenar todos los DataFrames en uno solo para el análisis global
df_total = pd.concat(lista_de_dfs, ignore_index=True)

print("--- Calculando ventas por categoría de producto ---")

# Asegurarse de que la columna 'Precio' sea numérica
df_total['Precio'] = pd.to_numeric(df_total['Precio'], errors='coerce')

# Eliminar filas con valores nulos en 'Categoría del Producto' o 'Precio'
df_total.dropna(subset=['Categoría del Producto', 'Precio'], inplace=True)

# Calcular el total de ventas (facturación) por cada categoría de producto
ventas_por_categoria = df_total.groupby('Categoría del Producto')['Precio'].sum().sort_values(ascending=False)

print("\nResumen de Ventas por Categoría:")
print(ventas_por_categoria.apply(lambda x: f"${x:,.2f}").to_string())

print("\n" + "="*60 + "\n") # Separador para mejor legibilidad

# Determinar la categoría más y menos popular (basado en facturación)
if not ventas_por_categoria.empty:
    categoria_mas_popular = ventas_por_categoria.index[0]
    ventas_mas_popular = ventas_por_categoria.iloc[0]

    categoria_menos_popular = ventas_por_categoria.index[-1]
    ventas_menos_popular = ventas_por_categoria.iloc[-1]

    print(f"La **categoría MÁS popular** (con más facturación) es: **{categoria_mas_popular}** con un total de **${ventas_mas_popular:,.2f}**")
    print(f"La **categoría MENOS popular** (con menos facturación) es: **{categoria_menos_popular}** con un total de **${ventas_menos_popular:,.2f}**")
else:
    print("No se encontraron datos de ventas válidos para analizar por categoría.")

print("\n" + "="*60 + "\n")

--- Calculando ventas por categoría de producto ---

Resumen de Ventas por Categoría:
Categoría del Producto
Electrónicos               $1,660,576,500.00
Electrodomésticos          $1,324,751,100.00
Muebles                      $757,661,000.00
Instrumentos musicales       $348,772,600.00
Deportes y diversión         $142,977,700.00
Juguetes                      $73,604,400.00
Artículos para el hogar       $57,579,800.00
Libros                        $37,696,100.00


La **categoría MÁS popular** (con más facturación) es: **Electrónicos** con un total de **$1,660,576,500.00**
La **categoría MENOS popular** (con menos facturación) es: **Libros** con un total de **$37,696,100.00**




# 3. Calificación promedio de la tienda


In [7]:
import pandas as pd

# URLs de tus bases de datos
url = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_1%20.csv"
url2 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_2.csv"
url3 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_3.csv"
url4 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_4.csv"

# Cargar los datos de cada tienda en DataFrames de Pandas
tienda1_df = pd.read_csv(url)
tienda2_df = pd.read_csv(url2)
tienda3_df = pd.read_csv(url3)
tienda4_df = pd.read_csv(url4)

# Diccionario para almacenar los DataFrames y facilitar la iteración
tiendas_data = {
    "Tienda 1": tienda1_df,
    "Tienda 2": tienda2_df,
    "Tienda 3": tienda3_df,
    "Tienda 4": tienda4_df
}

# Diccionario para almacenar la calificación promedio de cada tienda
calificacion_promedio_por_tienda = {}

print("--- Calculando la calificación promedio por tienda ---")
for nombre_tienda, df in tiendas_data.items():
    # Asegúrate de que la columna 'Calificación' exista y sea numérica
    if 'Calificación' in df.columns:
        # Convertir a numérico, forzando errores a NaN, y luego eliminar NaN antes de calcular la media
        df['Calificación'] = pd.to_numeric(df['Calificación'], errors='coerce')
        # Calcular el promedio de las calificaciones no nulas
        calificacion_promedio = df['Calificación'].mean()
        if pd.notna(calificacion_promedio): # Asegurarse de que el promedio no sea NaN (si todas las calificaciones son NaN)
            calificacion_promedio_por_tienda[nombre_tienda] = calificacion_promedio
            print(f"Calificación promedio para **{nombre_tienda}**: {calificacion_promedio:.2f}")
        else:
            print(f"Advertencia: No hay calificaciones válidas para {nombre_tienda}.")
            calificacion_promedio_por_tienda[nombre_tienda] = 0 # Asignar 0 si no hay calificaciones válidas
    else:
        print(f"Advertencia: La columna 'Calificación' no se encontró en los datos de {nombre_tienda}.")
        calificacion_promedio_por_tienda[nombre_tienda] = 0 # Asignar 0 si no se encuentra la columna

print("\n" + "="*60 + "\n") # Separador para mejor legibilidad

# Filtrar tiendas con calificaciones para evitar errores
if calificacion_promedio_por_tienda:
    tiendas_con_calificacion = {k: v for k, v in calificacion_promedio_por_tienda.items() if v > 0}

    if tiendas_con_calificacion:
        # Encontrar la tienda con más y menos calificación promedio
        tienda_mas_calificada = max(tiendas_con_calificacion, key=tiendas_con_calificacion.get)
        tienda_menos_calificada = min(tiendas_con_calificacion, key=tiendas_con_calificacion.get)

        print(f"La tienda con la **MÁS ALTA calificación promedio** es: **{tienda_mas_calificada}** con **{calificacion_promedio_por_tienda[tienda_mas_calificada]:.2f}** estrellas.")
        print(f"La tienda con la **MÁS BAJA calificación promedio** es: **{tienda_menos_calificada}** con **{calificacion_promedio_por_tienda[tienda_menos_calificada]:.2f}** estrellas.")
    else:
        print("No se encontraron calificaciones válidas para ninguna tienda.")
else:
    print("No se encontraron datos de calificaciones para analizar.")

# Opcional: Visualización con Pandas para un resumen tabular
print("\n" + "="*60 + "\n") # Separador
print("Resumen de Calificaciones Promedio (Tabla):")
df_calificaciones = pd.DataFrame(list(calificacion_promedio_por_tienda.items()), columns=['Tienda', 'Calificación Promedio'])
print(df_calificaciones.sort_values(by='Calificación Promedio', ascending=False).to_string(index=False))

--- Calculando la calificación promedio por tienda ---
Calificación promedio para **Tienda 1**: 3.98
Calificación promedio para **Tienda 2**: 4.04
Calificación promedio para **Tienda 3**: 4.05
Calificación promedio para **Tienda 4**: 4.00


La tienda con la **MÁS ALTA calificación promedio** es: **Tienda 3** con **4.05** estrellas.
La tienda con la **MÁS BAJA calificación promedio** es: **Tienda 1** con **3.98** estrellas.


Resumen de Calificaciones Promedio (Tabla):
  Tienda  Calificación Promedio
Tienda 3               4.048326
Tienda 2               4.037304
Tienda 4               3.995759
Tienda 1               3.976685


# 4. Productos más y menos vendidos

In [8]:
import pandas as pd

# URLs de tus bases de datos
url = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_1%20.csv"
url2 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_2.csv"
url3 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_3.csv"
url4 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_4.csv"

# Cargar los datos de cada tienda en DataFrames de Pandas
tienda1_df = pd.read_csv(url)
tienda2_df = pd.read_csv(url2)
tienda3_df = pd.read_csv(url3)
tienda4_df = pd.read_csv(url4)

# Diccionario para almacenar los DataFrames y facilitar la iteración
tiendas_data = {
    "Tienda 1": tienda1_df,
    "Tienda 2": tienda2_df,
    "Tienda 3": tienda3_df,
    "Tienda 4": tienda4_df
}

print("--- Análisis de Productos Más y Menos Vendidos por Tienda (basado en frecuencia) ---")

for nombre_tienda, df in tiendas_data.items():
    print(f"\n===== Análisis para **{nombre_tienda}** =====")

    # Asegúrate de que la columna 'Producto' exista
    if 'Producto' in df.columns:
        # Contar la frecuencia de cada producto
        conteo_productos = df['Producto'].value_counts()

        if not conteo_productos.empty:
            # Producto más vendido
            producto_mas_vendido = conteo_productos.index[0]
            frecuencia_mas_vendido = conteo_productos.iloc[0]

            # Producto menos vendido
            # Si hay muchos productos con la misma frecuencia mínima, toma el primero que aparece
            producto_menos_vendido = conteo_productos.index[-1]
            frecuencia_menos_vendido = conteo_productos.iloc[-1]

            print(f"  El producto **MÁS vendido** es: **{producto_mas_vendido}** con **{frecuencia_mas_vendido}** unidades vendidas.")
            print(f"  El producto **MENOS vendido** es: **{producto_menos_vendido}** con **{frecuencia_menos_vendido}** unidades vendidas.")

            # Opcional: Mostrar los 5 productos más y menos vendidos para un contexto más amplio
            print("\n  Top 5 productos más vendidos:")
            print(conteo_productos.head(5).to_string())
            print("\n  Top 5 productos menos vendidos:")
            print(conteo_productos.tail(5).to_string())

        else:
            print("  No se encontraron datos de productos para analizar en esta tienda.")
    else:
        print(f"  Advertencia: La columna 'Producto' no se encontró en los datos de {nombre_tienda}.")

print("\n" + "="*80 + "\n")

--- Análisis de Productos Más y Menos Vendidos por Tienda (basado en frecuencia) ---

===== Análisis para **Tienda 1** =====
  El producto **MÁS vendido** es: **Microondas** con **60** unidades vendidas.
  El producto **MENOS vendido** es: **Celular ABXY** con **33** unidades vendidas.

  Top 5 productos más vendidos:
Producto
Microondas          60
TV LED UHD 4K       60
Armario             60
Secadora de ropa    58
Mesa de noche       56

  Top 5 productos menos vendidos:
Producto
Ciencia de datos con Python    39
Pandereta                      36
Olla de presión                35
Auriculares con micrófono      33
Celular ABXY                   33

===== Análisis para **Tienda 2** =====
  El producto **MÁS vendido** es: **Iniciando en programación** con **65** unidades vendidas.
  El producto **MENOS vendido** es: **Juego de mesa** con **32** unidades vendidas.

  Top 5 productos más vendidos:
Producto
Iniciando en programación    65
Microondas                   62
Batería           

# 5. Envío promedio por tienda

In [10]:
import pandas as pd

# URLs de tus bases de datos
url = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_1%20.csv"
url2 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_2.csv"
url3 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_3.csv"
url4 = "https://raw.githubusercontent.com/alura-es-cursos/challenge1-data-science-latam/refs/heads/main/base-de-datos-challenge1-latam/tienda_4.csv"

# Cargar los datos de cada tienda en DataFrames de Pandas
tienda1_df = pd.read_csv(url)
tienda2_df = pd.read_csv(url2)
tienda3_df = pd.read_csv(url3)
tienda4_df = pd.read_csv(url4)

# Diccionario para almacenar los DataFrames y facilitar la iteración
tiendas_data = {
    "Tienda 1": tienda1_df,
    "Tienda 2": tienda2_df,
    "Tienda 3": tienda3_df,
    "Tienda 4": tienda4_df
}

# Diccionario para almacenar el costo promedio de envío por tienda
costo_promedio_envio_por_tienda = {}

print("--- Calculando el costo promedio de envío por tienda ---")
for nombre_tienda, df in tiendas_data.items():
    # Asegúrate de que la columna 'Costo de envío' exista
    if 'Costo de envío' in df.columns:
        # Convertir a numérico, forzando errores a NaN, y luego calcular la media.
        # Es importante manejar los nulos para un cálculo preciso del promedio.
        df['Costo de envío'] = pd.to_numeric(df['Costo de envío'], errors='coerce')
        costo_promedio = df['Costo de envío'].mean()

        if pd.notna(costo_promedio): # Asegurarse de que el promedio no sea NaN (si todas las entradas son NaN)
            costo_promedio_envio_por_tienda[nombre_tienda] = costo_promedio
            print(f"Costo promedio de envío para **{nombre_tienda}**: ${costo_promedio:,.2f}")
        else:
            print(f"Advertencia: No hay costos de envío válidos para **{nombre_tienda}**.")
            costo_promedio_envio_por_tienda[nombre_tienda] = 0 # Asignar 0 si no hay costos válidos
    else:
        print(f"Advertencia: La columna 'Costo de envío' no se encontró en los datos de **{nombre_tienda}**.")
        costo_promedio_envio_por_tienda[nombre_tienda] = 0 # Asignar 0 si no se encuentra la columna

print("\n" + "="*60 + "\n") # Separador para mejor legibilidad

# Filtrar tiendas con costos de envío para evitar errores
if costo_promedio_envio_por_tienda:
    tiendas_con_costo_envio = {k: v for k, v in costo_promedio_envio_por_tienda.items() if v > 0}

    if tiendas_con_costo_envio:
        # Encontrar la tienda con el costo de envío promedio más alto y más bajo
        tienda_mayor_costo_envio = max(tiendas_con_costo_envio, key=tiendas_con_costo_envio.get)
        tienda_menor_costo_envio = min(tiendas_con_costo_envio, key=tiendas_con_costo_envio.get)

        print(f"La tienda con el **MAYOR costo promedio de envío** es: **{tienda_mayor_costo_envio}** con un promedio de **${costo_promedio_envio_por_tienda[tienda_mayor_costo_envio]:,.2f}**.")
        print(f"La tienda con el **MENOR costo promedio de envío** es: **{tienda_menor_costo_envio}** con un promedio de **${costo_promedio_envio_por_tienda[tienda_menor_costo_envio]:,.2f}**.")
    else:
        print("No se encontraron costos de envío válidos para ninguna tienda.")
else:
    print("No se encontraron datos de costos de envío para analizar.")

# Opcional: Visualización con Pandas para un resumen tabular
print("\n" + "="*60 + "\n") # Separador
print("Resumen de Costos Promedio de Envío (Tabla):")
df_costos_envio = pd.DataFrame(list(costo_promedio_envio_por_tienda.items()), columns=['Tienda', 'Costo Promedio Envío'])
print(df_costos_envio.sort_values(by='Costo Promedio Envío', ascending=False).to_string(index=False))

--- Calculando el costo promedio de envío por tienda ---
Costo promedio de envío para **Tienda 1**: $26,018.61
Costo promedio de envío para **Tienda 2**: $25,216.24
Costo promedio de envío para **Tienda 3**: $24,805.68
Costo promedio de envío para **Tienda 4**: $23,459.46


La tienda con el **MAYOR costo promedio de envío** es: **Tienda 1** con un promedio de **$26,018.61**.
La tienda con el **MENOR costo promedio de envío** es: **Tienda 4** con un promedio de **$23,459.46**.


Resumen de Costos Promedio de Envío (Tabla):
  Tienda  Costo Promedio Envío
Tienda 1          26018.609580
Tienda 2          25216.235693
Tienda 3          24805.680373
Tienda 4          23459.457167


Estimado Cliente,

Es un placer presentarle un análisis consolidado del rendimiento de sus cuatro tiendas, con el objetivo de identificar áreas de oportunidad y tomar decisiones estratégicas.

Análisis de Rendimiento de Tiendas
Facturación General
Comenzamos con un resumen de la facturación global de sus tiendas:


Tienda 1: $1,150,880,400.00

Tienda 2: $1,116,343,500.00

Tienda 3: $1,098,019,600.00

Tienda 4: $1,038,375,700.00


La facturación total combinada de las cuatro tiendas asciende a $4,403,619,200.00.

Podemos observar claramente que la Tienda 1 es la que genera la mayor facturación, alcanzando los $1,150,880,400.00. En contraste, la Tienda 4 es la que presenta la menor facturación, con un total de $1,038,375,700.00.

Calificación Promedio de Clientes
La percepción de los clientes es vital. A continuación, el promedio de calificaciones por tienda:

Tienda 1: 3.98 estrellas

Tienda 2: 4.04 estrellas

Tienda 3: 4.05 estrellas

Tienda 4: 4.00 estrellas

La Tienda 3 destaca con la calificación promedio más alta (4.05 estrellas), lo que indica una mayor satisfacción general de sus clientes. Por otro lado, la Tienda 1, a pesar de ser la de mayor facturación, registra la calificación promedio más baja (3.98 estrellas). Esta es un área crucial a investigar.

Costo Promedio de Envío
Los costos operativos son un factor importante en la rentabilidad:

Tienda 1: $26,018.61

Tienda 2: $25,216.24

Tienda 3: $24,805.68

Tienda 4: $23,459.46

La Tienda 1 presenta el costo promedio de envío más alto, con $26,018.61, mientras que la Tienda 4 tiene el costo promedio de envío más bajo, con $23,459.46.

Recomendaciones Estratégicas
Basándonos en la información proporcionada, la Tienda 4 es la que actualmente presenta el menor desempeño en términos de facturación. Aunque tiene el menor costo promedio de envío, su bajo volumen de ingresos la sitúa en una posición vulnerable. Si el objetivo es optimizar el portafolio de tiendas y se considera la posibilidad de desinvertir en alguna, la Tienda 4 sería la principal candidata a revisar cuidadosamente.

No obstante, antes de tomar una decisión de venta, es fundamental profundizar en algunos aspectos:
Análisis de Rentabilidad Real: La facturación es solo una parte de la ecuación. Para determinar el "lucro" real (beneficio neto), necesitaríamos información sobre los costos operativos de cada tienda (alquiler, salarios, servicios, marketing, etc.), no solo los de envío. Una tienda con menor facturación podría ser más rentable si sus costos son significativamente más bajos.

Potencial de Crecimiento: ¿Existen factores externos (ubicación, competencia local, tamaño del mercado) que limiten el crecimiento de la Tienda 4 o que puedan ser mejorados con estrategias específicas?

Calificaciones y Experiencia del Cliente: La Tienda 4 tiene una calificación promedio de 4.00, que es buena, lo que sugiere que sus clientes actuales están satisfechos. Esto podría ser una base sólida para implementar estrategias de crecimiento.

Productos Vendidos:

Tienda 4: Su producto más vendido es "Cama box", y el menos vendido es "Guitarra eléctrica".

La categoría general menos popular es "Libros", con solo $37,696,100.00 en facturación, mientras que "Electrónicos" es la más fuerte, con $1,660,576,500.00. Esto podría indicar oportunidades para ajustar el inventario de la Tienda 4 si se enfoca en categorías de menor demanda a nivel general.

**Conclusión Preliminar:**

Aunque la Tienda 4 es la que registra la menor facturación, una decisión de "lucro" o "venta" debe ir más allá de los ingresos brutos. Le sugerimos realizar un análisis de costos y márgenes operativos detallados por cada tienda para tener una visión completa de su rentabilidad neta. Posteriormente, podríamos explorar estrategias específicas para potenciar la Tienda 4 o, si es el caso, justificar una desinversión con datos de rentabilidad más sólidos.

Estamos a su disposición para discutir estos hallazgos y planificar los siguientes pasos.